csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -8,10 +8,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0;net8.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0;net472</TargetFrameworks>
```

Comment:
We still want to support net462 in the core Arrow project because it's not yet at end-of-life. The change to net472 should be in Apache.Arrow.Flight.csproj.